diff --git a/node_modules/vuepress-theme-yuu/components/Navbar.vue b/node_modules/vuepress-theme-yuu/components/Navbar.vue
index a57a168..1b82d05 100644
--- a/node_modules/vuepress-theme-yuu/components/Navbar.vue
+++ b/node_modules/vuepress-theme-yuu/components/Navbar.vue
@@ -22,7 +22,7 @@
 			<div v-if="yuuConfig.extraOptions && yuuConfig.extraOptions.before" class="user-options-before">
 				<component :is="yuuConfig.extraOptions.before" @hook:mounted="handleLinksWrapWidth()" />
 			</div>
-			<user-settings />
+      <!-- <user-settings /> -->
 			<div v-if="yuuConfig.extraOptions && yuuConfig.extraOptions.after" class="user-options-after">
 				<component :is="yuuConfig.extraOptions.after" @hook:mounted="handleLinksWrapWidth()" />
 			</div>
diff --git a/node_modules/vuepress-theme-yuu/mixins/themeHandler.js b/node_modules/vuepress-theme-yuu/mixins/themeHandler.js
index 27e008e..b3f88d3 100644
--- a/node_modules/vuepress-theme-yuu/mixins/themeHandler.js
+++ b/node_modules/vuepress-theme-yuu/mixins/themeHandler.js
@@ -6,8 +6,8 @@ export default {
 				return localStorage.setItem('dark-theme', true)
 			}
 
-			document.body.classList.remove('yuu-theme-dark')
-			localStorage.setItem('dark-theme', false)
+			// document.body.classList.remove('yuu-theme-dark')
+			// localStorage.setItem('dark-theme', false)
 		},
 		setIgnoreThemes() {
 			this.setPageTheme()
diff --git a/node_modules/vuepress-theme-yuu/plugins/lifecycle.js b/node_modules/vuepress-theme-yuu/plugins/lifecycle.js
index 5e78e0c..b65ea6e 100644
--- a/node_modules/vuepress-theme-yuu/plugins/lifecycle.js
+++ b/node_modules/vuepress-theme-yuu/plugins/lifecycle.js
@@ -34,7 +34,7 @@ export default {
 
 		this.$root.$yuu = {
 			colorTheme: 'default',
-			darkTheme: false,
+			darkTheme: true,
 			ignoreThemes: false,
 			userTheme: undefined,
 		}
@@ -45,16 +45,13 @@ export default {
 		const userConfig = {
 			colorTheme,
 			darkTheme,
-			ignoreThemes: yuuConfig.disableThemeIgnore ? false : localStorage.getItem('ignore-themes') === 'true',
-			userTheme: localStorage.getItem('color-theme'),
+			ignoreThemes: false,
+			userTheme: 'green',
 		}
 
 		if (yuuConfig.disableDarkTheme !== true) {
-			if (yuuConfig.defaultDarkTheme === true && !localStorage.getItem('dark-theme')) {
-				localStorage.setItem('dark-theme', true)
-			}
-
-			userConfig.darkTheme = localStorage.getItem('dark-theme') === 'true'
+			localStorage.setItem('dark-theme', true)
+			userConfig.darkTheme = true
 		}
 
 		if (yuuConfig.defaultColorTheme !== 'default' && !userConfig.userTheme) {
